modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -200,39 +220,48 @@ function getRedeemIndex(keyCombinations: [Buffer, Buffer][], signer: Buffer, cos
   throw new Error(`could not find singer/cosigner combination`);
 }
 
-export function createPaymentP2tr(
+function createPaymentP2trCommon(
+  scriptType: 'p2tr' | 'p2trMusig2',
```

Comment:
NIT: this type is being repeated a lot, Can we have a type for it?

Reply:
Sure. This code will be revisited with few more PRs. It will be done along with them.

Reply:
IMO it's fine as it is - a new type adds an extra level of indirection here with little benefit